lib/internal/eachOfLimit.js

```diff
@@ -15,6 +17,9 @@ export default (limit) => {
         if (!obj) {
             return callback(null);
         }
+        if (isAsyncGenerator(obj)) {
```

Comment:
This is a minor thing, but should this be expanded to support async iterators in general? Something similar to how we currently have `getIterator` except with `Symbol.asyncIterator`.

Reply:
Ah, good point, didnt know about that detail of the spec.